Scripts/localize.py

```diff
@@ -121,7 +121,7 @@ def localize(path):
     old = original + '.old'
     new = original + '.new'
 
-    genstrings_cmd = 'genstrings -q -o "%s" `find ./Simplenote ./External/DTPinLock -name "*.m" -o -name "*.swift"`'
+    genstrings_cmd = 'genstrings -q -o "%s" `find ./Simplenote -name "*.m" -o -name "*.swift"`'
```

Comment:
I noticed a log about `DTPinLock` not being found in the `code_freeze` output. The library was removed in f8e00ec.

Reply:
Nice